File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.py
def solution(myString, pat):
    count = 0 #빈도수 리턴값
    index = 0
    
    while True:
        index = myString.find(pat,index)
        if index == -1:
            break
        count +=1
        index +=1
        
    return count

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java
class Solution {
    public int solution(String myString, String pat) {
        int index = 0; // indexOf에 사용할 인덱스 표시
        int count = 0; // 빈도수 리턴 값 저장
        
        while((index=myString.indexOf(pat,index)) != -1){
            count++;
            index += 1;
        }
        
        return count;
    }
}